backend/src/common/helpers/tokenHelper.ts

```diff
@@ -9,7 +9,7 @@ export const createRefreshToken = (id: string): string => jwt.sign({ id }, secre
 
 // eslint-disable-next-line no-return-await
 export const verifyToken = async (token: string) => await new Promise(resolve => {
-  jwt.verify(token, secret, (_err, decoded) => resolve(!!decoded));
+  jwt.verify(token, secret, (_err: any, decoded: any) => resolve(!!decoded));
```

Comment:
it was better before you explicitly set `any` because the parameters had implicit types (you can hover the cursor over them and see)